node/src/Pos/Wallet/Web/ClientTypes/Instances.hs

```diff
@@ -120,6 +108,19 @@ instance Buildable CAccountInit where
         caInitWId
         caInitMeta
 
+instance Buildable NewBatchPayment where
+    build NewBatchPayment{..} =
+        bprint ("{ from="%build
+                %" to="%(later mapBuilder)
+                %" inputSelectionPolicy="%build
+                %" }")
+        npbFrom
+        npbTo
+        npbInputSelectionPolicy
+
+instance Buildable (SecureLog NewBatchPayment) where
+    build = buildUnsecure
```

Comment:
Not good, this will be displayed in public logs and we don't want to see account/address ids there

Reply:
Got it. Added to todo list

Reply:
Fixed